database/migrations/2022_08_27_110209_drop_admin_user_config.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class DropAdminUserConfig extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		DB::table('configs')->where('key', '=', 'username')
+		->orWhere('key', '=', 'password')->delete();
```

Comment:
```suggestion
		DB::table('configs')
			->whereIn('key', ['username', 'password'])
			->delete();
```

This is nitpicking, but `whereIn` is more efficient than `orWhere`, especially with the silly MySQL query planner. `whereIn` allows to use an index, but if MySQL finds an `OR` in a query it stops trying to optimize the query or using an index, because MySQL cannot deal with `OR` in queries. :disappointed: 

But surely, this is irrelevant for a query which is only run at most once. :smile: 

Moreover, I believe the code is more succinct and cleaner.